usaspending_api/reporting/v2/views/agencies/overview.py

```diff
@@ -50,8 +52,14 @@ def get_agency_overview(self):
             Q(fiscal_period=self.fiscal_period),
         ]
         result_list = (
-            ToptierAgency.objects.account_agencies()
-            .filter(*agency_filters)
+            ToptierAgency.objects.annotate(
+                has_submission=Exists(
```

Comment:
Not sure if this is the correct logic for limiting, but we have another ticket that covers limiting to only those agencies that have published a DABS. I imagine that ticket will review this more closely.